test/controllers/wellcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
I think this test is unnecesary, Please delete it. We don't work with unit test Yet, We are going to work with Rspec.